### PR TITLE
MINOR: Make private class FetchManagerMetrics as a static private class with

### PR DESCRIPTION
**Pull request by rajatvig**
reduced visibility for members in Fetcher.

Use Collections.singletonList instead of Arrays.asList in the Unit Test

### CODE REVIEW

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java [hunk 1]
```diff
@@ -563,19 +563,15 @@ public FetchManagerMetrics(Metrics metrics, String metricGrpPrefix, Map<String,
...
-            if (recordsFetched == null)
-                recordsFetched = this.metrics.sensor(name);
-            recordsFetched.record(records);
+            getSensor("topic." + topic + ".bytes-fetched").record(bytes);
+            getSensor("topic." + topic + ".records-fetched").record(records);
+        }
+
+        private Sensor getSensor(String name1) {
+            Sensor topic = this.metrics.getSensor(name1);
+            if (topic == null)
```
Comment (eribeiro) [inline clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java]: nit: those lines can be rewritten (simplified?) as:
`return (topic != null) ? topic: metrics.sensor(name1);`

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java [hunk 2]
```diff
@@ -563,19 +563,15 @@ public FetchManagerMetrics(Metrics metrics, String metricGrpPrefix, Map<String,
...
-            name = "topic." + topic + ".records-fetched";
-            Sensor recordsFetched = this.metrics.getSensor(name);
-            if (recordsFetched == null)
-                recordsFetched = this.metrics.sensor(name);
-            recordsFetched.record(records);
+            getSensor("topic." + topic + ".bytes-fetched").record(bytes);
+            getSensor("topic." + topic + ".records-fetched").record(records);
+        }
+
+        private Sensor getSensor(String name1) {
```
Comment (eribeiro) [inline clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java]: nit: `name1` is strange, why not just `name`?

**Comment by rajatvig** [comment]
Test failure seems unrelated to the change.

**Comment by guozhangwang** [comment]
@rajatvig Would you like to continue working on this PR and rebasing? If yes I would love to review it (you can assign myself to the reviewers list on the top right).

**Comment by rajatvig** [comment]
This seems not required any more.

**Comment by b-goyal** [comment]
reduced visibility for members in Fetcher.

Use Collections.singletonList instead of Arrays.asList in the Unit Test

**Pull request closed without merging**